components/homme/CMakeLists.txt

```diff
@@ -159,11 +159,14 @@ IF (HOMME_ENABLE_COMPOSE)
   SET (HOMME_USE_KOKKOS TRUE)
 ENDIF ()
 
-IF (HOMME_USE_CXX)
-  MESSAGE(STATUS "This configuration of HOMME requires a C++ compiler")
-  ENABLE_LANGUAGE(CXX)
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-ENDIF ()
+IF (NOT "${PROJECT_NAME}" STREQUAL "SCREAM")
```

Comment:
I think this may be an issue down the road. E.g., when scream is merged in e3sm, I'm not sure if this check would fail. Also, it would be perfectly legal for homme devs to declare homme as a project, regardless of whether there is a host project or not (cmake lets you do that, and it should arguably be done here).

- at the top of this file, set the bool HOMME_STANDALONE based on whether PROJECT_NAME is defined or not.
- in the rest of the file, use `if (HOMME_STANDALONE)` to isolate these chunks of code that should only be run in standalone mode.

Reply:
@bartgol , the situation is complicated. I think Homme wants to manage its own flags when it is a subcomponent of E3SM, right?

Reply:
Yes, but when building homme in EAM, the cmake system is not used. EAM uses homme via the perl/bash/python scripts.

Reply:
@bartgol , thanks, I did not know that.